contracts/tools/check_aderyn.sh

```diff
@@ -11,6 +11,10 @@ SEVERITIES=(critical high medium)
 # List of vulnerability titles to ignore
 IGNORE_TITLES=("Centralization Risk for trusted owners")
 
+# Specific vulnerabilities to ignore with path and line number
+declare -A IGNORE_SPECIFIC
+IGNORE_SPECIFIC["src/lib/LibDiamond.sol:204:Unprotected initializer"]=1
```

Comment:
This line number is certainly going to change :-/ Does the Aderyn report not emit a symbol that we can match against?

Reply:
I don't see any convenient way for allow listing a specific failure. Aderyn provides a way to ignore a specific file, but that feels like a way to set ourselves up for problems down the line

Reply:
> This line number is certainly going to change

I'm curious what you mean by that because I'm seeing a really confusing pattern at the moment. On my desktop the error claims to be for line 204 but the github action claims the error is on line 203. 

also because this specific error that i'm filtering is in the file `src/lib/LibDiamond.sol` I think that is something we will touch relatively infrequently